### PR TITLE
feat: home page

### PR DESCRIPTION
**Pull request by partha8**
Hey everyone, added the homepage of my quiz app. Please share your views. Thank you

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,82 @@
...
+                    <div class="desc">
+                        <h4 class="card-title">Famous hollywood characters</h4>
+                    </div>
+                </a>
+            </article>
+
+            <article class="category">
+                <a href="#">
+                    <img class="category-img" src="./assets/holding-al-pacino-best-and-worst-film-moments.jpg" alt="">
+                    <div class="desc">
```
Comment (Maianki) [inline index.html]: you can give values to alt attributes.

**Review by Maianki — APPROVED** [review]
Suggestion: You can compress your images as one of them is above 1 mb. Everything else looks good.

File: index.html (hunk 1 above)
Comment (partha8) [reply index.html]: Oh yea, totally forgot about them. thank you!

**Comment by partha8** [comment]
> Suggestion: You can compress your images as one of them is above 1 mb. Everything else looks good.

Yeah, I was actually thinking about it too.

**Review by aayushjain-code — COMMENTED** [review]
Everything looks, fine bro 
just a suggestion try to decrease the size of the sub heading a bit.

**Review by ParthParolekar — APPROVED** [review]
Everything looks good to me!

**Pull request merged**